sympy/combinatorics/fp_groups.py

```diff
@@ -494,6 +496,97 @@ def elements(self):
         P, T = self._to_perm_group()
         return T.invert(P._elements)
 
+    @property
+    def is_polycyclic(self):
+        if self.is_solvable:
+            return True
+        return None
+
+    def is_subgroup(self, other):
+        """
+        Return ``True`` if all elements of ``self`` belong to ``other``.
+        """
+        _grp1 = self._to_perm_group()[0]
+        _other = self._to_perm_group()[0]
+        return _grp1.is_subgroup(_other, strict=False)
+
+    def compute_polycyclic_series(self):
+        '''
+        ToDo - Update docstring.
+        '''
+        if not self.is_polycyclic:
+            raise NotImplementedError("Polycyclic methods are only"
+                                        "implemented for FpGroups")
```

Comment:
That error message doesn't match the reason it is being raised. The reason is that the group is not known to be polycyclic. So a better message, perhaps, would be `No methods currently implemented to check if the given group is polycyclic` or something along those lines.